chacha20/src/rng.rs

```diff
@@ -735,7 +735,7 @@ pub(crate) mod tests {
         let mut rng1 = ChaChaRng::from_seed(seed);
         assert_eq!(rng1.next_u32(), 137206642);
 
-        let mut rng2 = ChaChaRng::from_rng(rng1);
+        let mut rng2 = ChaChaRng::from_rng(&mut rng1);
```

Comment:
This looks like some kind of nightly-only failure: https://github.com/RustCrypto/stream-ciphers/actions/runs/13020926488/job/36321197288

That said, the new code is more explicit, rather than relying on the blanket impl.